core/src/main/java/com/linecorp/armeria/server/AggregatedHttpResponseHandler.java

```diff
@@ -57,7 +57,7 @@ final class AggregatedHttpResponseHandler extends AbstractHttpResponseHandler
 
     @Override
     public Void apply(@Nullable AggregatedHttpResponse response, @Nullable Throwable cause) {
-        final EventLoop eventLoop = reqCtx.eventLoop();
```

Comment:
Note: `apply0` will directly call `responseEncoder.write*` which should be called from the channel event loop

https://github.com/line/armeria/blob/e344c964345dfb5037c654b82709bfcaaea83264/core/src/main/java/com/linecorp/armeria/server/AbstractHttpResponseHandler.java#L185